app/src/main/java/com/example/reflector_android/Models/BlogPost.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.reflector_android.Models
+
+data class BlogPost(
```

Comment:
Style guide does not specify but I think this can be put all on one line like our other declarations.

Reply:
Don't need the file anymore so I deleted it